javascript-view-swapping/main.js

```diff
@@ -0,0 +1,27 @@
+var $tabContainer = document.querySelector('.tab-container');
+var $tab = document.querySelectorAll('.tab');
+var $view = document.querySelectorAll('.view');
+
+$tabContainer.addEventListener('click', function (event) {
+
+  if (event.target.matches('.tab')) {
+
+    for (var i = 0; i < $tab.length; i++) {
+      if ($tab[i] === event.target) {
+        $tab[i].className = 'tab active';
+      } else {
+        $tab[i].className = 'tab';
+      }
+    }
+  }
+  if (event) {
```

Comment:
What if you click next to the tabs but not on a specific tab? You'll find that the entire view section empties. This portion should only be done if the `event.target` was a `tab` as well